fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
There is _nothing_ hypothetical about the requirements in hand. We have resources in hand now that have attached to them a technical characterization, triples that describe the relationships to other resources, MODS in XML, triples that describe their state of preservation, etc.... the use of multiple descriptions per resource is _exactly_ what we need _today_. The fact that we have to work on the problem from several fronts at once is annoying. I get that. But it's not a reason to just not work on the problem. If you want to work on the "repo-side" part of multiple descriptions per resource so that this part doesn't seem disconnected, rock on.